src/app/component/AddressInput/AddressInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import AlgoliaPlaces from "algolia-places-react";
+import PropTypes from "prop-types";
+
+const AddressInput = (props) => {
+  const { stage, setStage, placeholder } = props;
+  const handleLocation = (query) => {
+    if (query.suggestion) {
+      const { value, latlng, county, countryCode } = query.suggestion;
+      setStage({
+        ...stage,
+        location: {
+          address: value,
+          lat: latlng.lat,
+          long: latlng.lng,
+          county,
+          countryCode,
+        },
+      });
+    }
+  };
+  return (
+    <AlgoliaPlaces
+      placeholder={placeholder || "Search address"}
+      name={stage}
+      options={{
+        appId: "plZ318O8ODTC",
+        apiKey: "b5e0781d289a9aa8edb37bf24aef874e", // only 1k per day
```

Comment:
Can we abstract this out into a constant somewhere? I know theres a few other places this is used.